docs/development/new-packages.md

```diff
@@ -89,6 +89,45 @@ If there are errors you might need to
 
 then restart the build.
 
+If the build succeeds you can try to load the package by
+
+1. If you haven't built Pyodide from source, download the
+   [most recent Pyodide release](https://github.com/pyodide/pyodide/releases) and extract it into the `dist` directory
```

Comment:
I find that step 1) confusing. I don't think it belongs here under `3. Building the package and investigating issues`. Everything in this part is about building the package so that it gets dropped into `pyodide/dist` with everything in there to run it. If I where to unpack the stable release in that directory, I would overwrite a) all the dependencies that were just compiled (if it works) with the versions from the tarball and b) also the package.json that would then again lack the new package.

If going through with forcing the new package build to also rebuild all dependencies, I think this should only list steps 2.-4.

But I still think a workflow were really only the new package is built and just using everything else from the stable release would be the easiest solution for people to try and build their package, if all dependencies are already in the stable release.

Reply:
> a workflow were really only the new package is built and just using everything else from the stable release would be the easiest solution

We are working toward this. Hopefully the next major release will include this feature.